File: src/ft.employee-management/ft.employee-management.Domain/Entities/Employee.cs
using ft.employee_management.Domain.Enums;

namespace ft.employee_management.Domain.Entities;

public class Employee : BaseDomainEntity
{
    public string? FirstName { get; set; }
    public string? LastName { get; set; }
    public string? Email { get; set; }
    public DateTime? DateOfBirth { get; set; }
    public DateTime HireDate { get; set; }
    public string? Gender { get; set; }
    public string? PhoneNumber { get; set; }
    public decimal Bonus { get; set; }
    public decimal Salary { get; set; }
    public EmployeeType EmployeeType { get; set; }
}

File: src/ft.web-ui/src/types/employee.ts
export type Employee = {
  id: number 
  firstName: string
  lastName: string
  dateOfBirth?: string | null
  gender: string
  email: string
  phoneNumber: string
  type: string
  hireDate: string
  bonus: number,
  salary: number
}
const testEmployee: Employee = {
  'id': 1002,
  'firstName': 'Nathnael',
  'lastName': 'Gebreselassie',
  'email': 'nathnael.gebreselassie@gmail.com',
  'dateOfBirth': null,
  'hireDate': '6/12/2023 12:00:00 AM',
  'gender': 'male',
  'phoneNumber': '0932152547',
  'bonus': 5000,
  'salary': 25000,
  'type': 'FullTime'
}